example/react-on-client/README.md

```diff
@@ -0,0 +1,34 @@
+# React on Client
+
+This is an example Pogo app with React client-side rendering.
+
+Based on [Simple Server](../simple-server), with the following changes:
+ - Added [`index.html`](./index.html), where the app will be rendered on the client
+ - Added a [`client`](./client) directory with frontend modules such as [`client/app.jsx`](./client/app.jsx)
+ - Added [`client/counter.jsx`](./client/counter.jsx), a counter component that uses `React.useState`
+
+## Run the example
+
+*Make sure [Deno](https://deno.land/) is installed and up to date.*
+
+### Remote
+
+The fastest way to run the example is to use its URL:
+
+```sh
+deno -A https://deno.land/x/pogo/example/react-on-client/run.ts
```

Comment:
This won't work without a build. Right?

Need to either:
A. Add the build steps here
B. Update `run.ts` to do the build steps automatically, for example with `await Deno.mkdir('build', { recursive : true })`

Reply:
A) I wasn't sure how to test this since it pulls from deno.land.
B) I didn't like the idea of moving the build step into run.ts as it is not best practice for servers to build source files on bootup.

I stripped the instructions for the remote example based on the above. But if you want A and know how to accomplish it please amend to this PR.